beard/metrics/text.py

```diff
@@ -162,3 +164,51 @@ def jaro_winkler(s1, s2, p=0.1):
             break
 
     return jaro_distance + p * common_prefix * (1 - jaro_distance)
+
+
+def levenshtein(a, b):
+    """Calculate the levenshtein distance between strings a and b.
+
+    Case sensitiveness is activated, meaning that uppercase letters
```

Comment:
I would prefer to keep a clear separation of concerns. If someone wants case insensitiveness, then this should be explicit by calling `.lower()` before calling this function. But this is just my opinion. If others feel it would be nice, then fine.